src/utils.ts

```diff
@@ -36,6 +37,16 @@ export function buildUrl(
   body.push(`URL:${event.url || document.URL}`);
   body.push(`DTSTART:${formatDate(event.startTime)}`);
   body.push(`SUMMARY:${event.title}`);
+  if (event.attendees) {
+    event.attendees.map(attendee => {
+      const regExp = attendee.match(/(?:"?([^"]*)"?\s)?(?:<?(.+@[^>]+)>?)/);
+      const name = regExp && regExp[1];
+      const email = regExp && regExp[2];
+      return { name, email };
+    }).forEach(({ name, email }) => {
+      body.push(`ATTENDEE;CN=${name};CUTYPE=INDIVIDUAL;PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:${email}`);
```

Comment:
It's a long string and not very easy to maintain and read. I personally would like to separate like this way. 

```js
body.push([
  'ATTENDEE'
  `CN=${name}`,
  'CUTYPE=INDIVIDUAL',
  'PARTSTAT=NEEDS-ACTION',
  'ROLE=REQ-PARTICIPANT',
  `RSVP=TRUE:mailto:${email}`
].join(';'));
```

Do you feel it's easier or not necessary?